cordova-ionic-phonegap-branch-deep-linking-attribution.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is an auto generated file from your environment- I recommend excluding this change then adding *.iml to the .gitignore